AmplifyPlugins/DataStore/AWSDataStoreCategoryPlugin/Storage/SQLite/Statement+Model.swift

```diff
@@ -75,7 +78,7 @@ extension Statement: StatementModelConvertible {
             guard let (schema, field) = columnMapping[column] else {
                 logger.debug("""
                 A column named \(column) was found in the result set but no field on
-                \(modelType.modelName) could be found with that name and it will be ignored.
+                \(modelSchema.name) could be found with that name and it will be ignored.
```

Comment:
This appears also not to be broken, but changed due to the fact that we are moving to passing in the schema instead of the type.